services/galley/src/Galley/Cassandra/Conversation.hs

```diff
@@ -266,7 +326,12 @@ toProtocol ::
 toProtocol Nothing _ _ _ = Just ProtocolProteus
 toProtocol (Just ProtocolProteusTag) _ _ _ = Just ProtocolProteus
 toProtocol (Just ProtocolMLSTag) mgid mepoch mcs =
-  ProtocolMLS <$> (ConversationMLSData <$> mgid <*> mepoch <*> mcs)
+  ProtocolMLS <$> (ConversationMLSData <$> mgid <*> mapEpoch mepoch <*> mcs)
+  where
+    -- If there is no epoch in the database, assume the epoch is 0 and this is a
+    -- self-conversation.
```

Comment:
We don't actually need to assume that it is a self-conversation, do we?

Reply:
I dropped that part of the comment. This was the first time (from this PR's point of view) that we started with this approach to MLS conversation creation, hence the comment does apply to self-conversations only.